docs/griptape-framework/structures/src/json_schema_rule_pydantic.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import annotations
+
+import pydantic
+
+from griptape.rules.json_schema_rule import JsonSchemaRule
+from griptape.structures import Agent
+
+
+class OutputFormat(pydantic.BaseModel):
+    answer: str
+    relevant_emojis: list[str]
+
+
+agent = Agent(rules=[JsonSchemaRule(OutputFormat.model_json_schema())])
+
+output = agent.run("What is the sentiment of this message?: 'I am so happy!'").output
+
+print(OutputFormat.model_validate_json(output.value))
```

Comment:
How about highlighting some of the sugar!

```py
from pydantic import BaseModel, Field

class SentimentAnalysis(BaseModel):
    sentiment: str = Field(description="Sentiment of the message, one of 'positive', 'neutral', 'negative'")
    relevant_emojis: list[str]

...

sentiment_analysis_text = ...

sentiment_analysis = SentimentAnalysis.model_validate_json(...)

# Autocomplete via dot notation 🤩 
print("sentiment: ", sentiment_analysis.sentiment)
print("relevant_emojis: ", sentiment_analysis.relevant_emojis)
```

Reply:
Sugar added!